### ISSUE TITLE
egs2/tedlium2 upload

### ISSUE THREAD

**Issue by flylili001** [espnet/espnet#2714]
I have run an example of tedlium2 in espnet2, Can I upload it to egs2?
And WER are:
tedlium2_dev | 8.9
tedlium2_test | 9.1

Thank you.

**Comment by b-flo** [comment]
Hi,

Thanks for the help!
I see a PR for Tedlium2 port was opened in the past (#1687) but it's now closed. @simpleoier What is the state of the recipe? I guess @flylili001 could make a new PR anyways?

**Comment by sw005320** [comment]
> I have run an example of tedlium2 in espnet2, Can I upload it to egs2?
> And WER are:
> tedlium2_dev | 8.9
> tedlium2_test | 9.1
> 
> Thank you.

Very good!!!

> Thanks for the help!
> I see a PR for Tedlium2 port was opened in the past (#1687) but it's now closed. @simpleoier What is the state of the recipe? I guess @flylili001 could make a new PR anyways?

Year, since @flylili001 did a great job, it would be great if @flylili001 makes PR for @simpleoier

**Comment by flylili001** [comment]
Thank you for reply. I am preparing the upload. But I found the difference of new version of espent in asr.sh:
inference_asr_model=valid.acc.ave.pth
in the previous version:
inference_asr_model=valid.acc.best.pth
the previous version setting could get better performance, could I update "asr.sh"?
thank you.

**Comment by sw005320** [comment]
You can specify it (adding an option of `--inference_asr_model valid.acc.best.pth`) in tedlium2 `run.sh` instead of updating `asr.sh` since `asr.sh` would affect all other recipes.

However, I'm curious about it because I always observe the improvement when we use the averaged model.
Maybe, we can continue the discussion at your PR, since I also want to take a look at the config files in more detail.

**Comment by flylili001** [comment]
I have one issue, the performances in different version are different:
v0.9.5: WER=(dev:10.3,test:9.9)
v0.9.0: WER=(dev:8.9, test:9.1)
I carefully compared the difference of two version, I found the default setting in asr.sh of v0.9.0 is inference_asr_model=valid.acc.best.pth, and v0.9.5 is inference_asr_model=valid.acc.avg.pth.
But in v0.9.5, I set inference_asr_model=valid.acc.best.pth, the performance is worse: WER=(dev:11.2,test:11.1).
I don't know why?
